contracts/p0/StRSR.sol

```diff
@@ -378,6 +382,20 @@ contract StRSRP0 is IStRSR, ComponentP0, EIP712Upgradeable {
             address account = accounts.at(i);
             delete withdrawals[account];
         }
+        emit AllUnstakingReset(era);
+    }
+
+    /// @custom:governance
+    /// Reset all stakes and advance era
+    function resetStakes() external governance {
+        uint192 stakeRate = FIX_ONE.div(exchangeRate());
```

Comment:
I think we can use `divuu(totalStaked, rsrBacking)`. Not 100% sure on the rounding, but I think this will work